### DISCUSSION TITLE
modify imported task formatting

### DISCUSSION THREAD

**Discussion by kfitz** [open-horizon-labs/obsidian-ics#65] [Ideas]
Hi! I so, so love this plugin -- thank you for building and sharing it. I have been manually transcribing my calendar events into my daily note each day, and this promises to streamline that in wonderful ways.

The one question I have is whether there might be a way to allow the user to control the ways imported tasks are presented. I list all my daily events in a callout, and then use Tasks to display all my to-dos for the day in a second callout. Tasks doesn't like that my ICS imported events look to it like tasks (because checkbox) but are in a callout, and gives me an error every time I try to mark one complete. I don't really need to mark them complete, except that Tasks will then gather them as incomplete tasks. So if I could just import them as unordered list items, that would be better.

Thanks again!

**Comment by FeralFlora** [comment]
@kfitz You can change the task format to a list format in the per-calendar settings.

**Comment by muness** [comment]
Thanks @FeralFlora ! I added that feature in [1.1.5](https://github.com/muness/obsidian-ics/releases/tag/1.1.5) last week for @kfitz but then forgot to update this discussion.

**Reply by kfitz** [reply]
Thank you!!!

**Reply by muness** [reply]
You are welcome! It makes me happy so many other people find this useful too. 😄
